numpy/lib/tests/test_npyio.py

```diff
@@ -0,0 +1,53 @@
+from numpy.testing import assert_, assert_raises
```

Comment:
test_io.py is testing another module than the module npyio, without importing the assert_ (or assert_equal) in the testing file test_npyio it didn't work. So we left this line in the file.